README.md

```diff
@@ -15,3 +15,14 @@ GET /wishlists | LIST | Show all wishlists
 GET /wishlists?q=querytext | QUERY | Search for a wishlist
 GET /wishlists/`<id>`?q=querytext | QUERY | Search for items in wishlist
 PUT /wishlists/`<id>`/items/`<itemid>`/add-to-cart | ACTION | Move an item from wishlist to the shopping cart
+
+To execute the service run:
+
+```
+vagrant up --provision
+vagrant ssh
+cd /service
```

Comment:
This will error out. We only need `/vagrant`

Reply:
No, it needs both, these were the instructions that I had in my local branch. /vagrant only won't work either.  These instructions are no longer valid.  
I am updating my local copy with all the changes and submit a new update later today.

Reply:
Okay. Thanks